pkg/clustertree/knode-manager/utils/util.go

```diff
@@ -9,6 +12,11 @@ func SetObjectGlobal(obj *metav1.ObjectMeta) {
 	obj.Annotations[KosmosGlobalLabel] = "true"
 }
 
+func FilterSecret(secretInSub *v1.Secret) error {
```

Comment:
FilterSecret could be under IsObjectGlobal.

Reply:
I deleted x FilterSecret and re-used TrimObjectMeta